lib/accessly/policy/base.rb

```diff
@@ -0,0 +1,270 @@
+module Accessly
+  module Policy
+    class Base
+
+      attr_reader :actor
+
+      def initialize(actor)
+        @actor = actor
+      end
+
+      def self.actions(actions)
+        _actions.merge!(actions)
+        actions.each do |action, action_id|
+          _define_action_methods(action, action_id)
+        end
+      end
+
+      def self.actions_on_objects(actions_on_objects)
+        _actions_on_objects.merge!(actions_on_objects)
+        actions_on_objects.each do |action, action_id|
+          _define_action_methods(action, action_id)
+        end
+      end
+
+      def self.namespace
+        String(self)
+      end
+
+      def namespace
+        self.class.namespace
+      end
+
+      def self.model_scope
+        puts caller
```

Comment:
Do we want to log this in some way? (or was this for general debugging)

Reply:
Whoops! This was for debugging. I'll remove.